src/api/app/helpers/webui/request_helper.rb

```diff
@@ -199,4 +200,18 @@ def next_prev_path(**opts)
       request_show_path(parameters)
     end
   end
+
+  def build_status_needs_attention?(buildresults)
+    buildresults.any?(&:unsuccessful_final_status?)
+  end
```

Comment:
Could this be moved to the BsRequestAction model?

It seems more natural for me to ask the action if any of its build results need attention... 
having the answer, you use the helpers to shape the output in the view.

Reply:
Yeah, makes sense. Let me change it